public_html/wp-content/themes/pattern-directory/src/components/active-filter-bar/index.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
> Are we keen on building out the components using JS or do we want to build them in PHP templates?

Since it's interactive, JS seems to give a better UX IMO, since it will be faster/smoother.

Reply:
Sounds good.